tree/ntupleutil/v7/src/RNTupleInspector.cxx

```diff
@@ -219,6 +220,53 @@ ROOT::Experimental::RNTupleInspector::GetColumnsByType(ROOT::Experimental::EColu
    return colIds;
 }
 
+void ROOT::Experimental::RNTupleInspector::PrintColumnTypeInfo(ENTupleInspectorPrintFormat format, std::ostream &output)
```

Comment:
For clarity, consider adding as documentation of this function samples of the output for both format (i.e. PR summary content + a csv example).